### PR TITLE
Published Icon

### LINKED ISSUE #4: Show share/publish icon in note list
<a href="https://github.com/mikejohnstn"></img></a> **Issue by [mikejohnstn](https://github.com/mikejohnstn)**
_Monday Jun 10, 2013 at 19:09 GMT_

---

### PR DESCRIPTION
**Pull request by jleandroperez**
### Details:
In this PR we're adding a "Published" Indicator in the right hand side of any note that has been made publicly available.

Closes #4 

cc @bummytime for code review
cc @SylvesterWilmott for design review!

Thank you both!!

### Testing:
1. Log into your account
2. Select any of your notes
3. Click over the Share Icon (Top Right actions, third one from the right)
4. Select `Publish to Web`
5. Publish the note!

Verify that published notes get an icon in the right hand side of their cells.

---

### Screenshots:

Screen Shot 2019-07-04 at 2 36 14 PM
Screen Shot 2019-07-04 at 2 36 24 PM

### CODE REVIEW

File: Simplenote/SPNoteCellView.m [hunk 1]
```diff
@@ -39,26 +41,27 @@ - (void)setBackgroundStyle:(NSBackgroundStyle)backgroundStyle
...
 }
 
 - (void)updatePreview
 {
-    NSColor *headlineColor = highlighted ? [self.theme colorForKey:@"tintColor"] : [self.theme colorForKey:@"noteHeadlineFontColor"];
-    NSColor *previewColor = highlighted ? [self.theme colorForKey:@"tintColor"] : [self.theme colorForKey:@"noteBodyFontPreviewColor"];
-    NSString *preview = [note.content length] == 0 ? @"New note..." : [note.content stringByGeneratingPreview];
+    NSColor *headlineColor = _highlighted ? [self.theme colorForKey:@"tintColor"] : [self.theme colorForKey:@"noteHeadlineFontColor"];
+    NSColor *previewColor = _highlighted ? [self.theme colorForKey:@"tintColor"] : [self.theme colorForKey:@"noteBodyFontPreviewColor"];
+    NSString *preview = [_note.content length] == 0 ? @"New note..." : [_note.content stringByGeneratingPreview];
```
Comment (bummytime) [inline Simplenote/SPNoteCellView.m]: I know this was existing code, but do you think we should use `NSLocalizedString` for `@"New note..."`?

Comment (jleandroperez) [reply Simplenote/SPNoteCellView.m]: ARGH didn't notice!!! of course we can, apologies abiout missing that!!

**Comment by SylvesterWilmott** [comment]
Looks good!

Let's make sure we also use the left margin on the icon here as well.

**Comment by jleandroperez** [comment]
@SylvesterWilmott Left padding increased to 16pt (same metric we use in iOS).

Just for sanity, does this look great to you? (Thank you!!!)

Screen Shot 2019-07-08 at 10 57 32 AM

**Comment by SylvesterWilmott** [comment]
Looks good. Although, I noticed this icon is at 24px while the others are at 22px. Can we just change the size from 24 to 22 to be consistent with the other icons?

**Comment by jleandroperez** [comment]
@SylvesterWilmott *actually* it's the same asset all over. It's 24x24pt in iOS as well, see the screenshot below:

Share iOS

**Comment by jleandroperez** [comment]
Okay! As discussed over Slack (for documentation purposes), asset resized to 22x22pt.

Thank you @SylvesterWilmott !!

Screen Shot 2019-07-08 at 11 32 36 AM
Screen Shot 2019-07-08 at 11 32 30 AM

**Pull request merged**